python/pip_install/private/test/BUILD

```diff
@@ -0,0 +1,14 @@
+load("@bazel_skylib//rules:diff_test.bzl", "diff_test")
+
+diff_test(
+    name = "srcs_diff_test",
+    failure_message = "Please run `bazel run @rules_rust//python/pip_install/private:srcs_module.install`",
```

Comment:
You should provide a description of the side effects of running that command as well.